executor/load_data_test.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package executor
+
+import (
+	"fmt"
+	"math"
+	"runtime"
+	"testing"
+
+	"github.com/pingcap/tidb/br/pkg/lightning/config"
+	"github.com/pingcap/tidb/expression"
+	"github.com/pingcap/tidb/parser"
+	"github.com/pingcap/tidb/parser/ast"
+	plannercore "github.com/pingcap/tidb/planner/core"
+	"github.com/pingcap/tidb/util/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoadDataWorker_initDefaultOptions(t *testing.T) {
```

Comment:
There's no naming style like this elsewhere?

Reply:
auto generated by goland

Reply:
https://github.com/pingcap/tidb/pull/41890/commits/4cd97a1d92efdd483861379cec8bc745b93a4ed1